### ISSUE TITLE
Casing of functions

### ISSUE THREAD

**Issue by ChrisLGardner** [Microsoft/DSCEA#59]
Hi

I'm wondering if there was a reason you chose the casing for functions as you did? The two obvious "issues" I see are the following:

* DSC used instead of Dsc, as seen in PSDesiredStateConfiguration module
* Use of a lower case letter for the first word following DSCEA, e.g. Start-DSCEAscan

Both of these diverge from the [PowerShell Style Guide](https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style%20Guide/Code%20Layout%20and%20Formatting.md#capitalization-conventions) and the (.Net Framework Capitalization Conventions)[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#Anchor_1].

**Comment by krjhitch** [comment]
I agree, it should be Start-DscEaScan and/or Get-DscEaPowerBiReport (or similar).  Will be altering this in the next release

**Comment by ChrisLGardner** [comment]
I think Start-DscEAScan and Get-DscEAPowerBIReport would be the correct capitalization, both EA and BI are two letter acronyms and conventions suggest they should be capitalized. 

Thanks.
